pyanaconda/payload/dnf/payload.py

```diff
@@ -2005,6 +2005,17 @@ def _write_dnf_repo(self, repo, repo_path):
             if ks_repo.excludepkgs:
                 f.write("exclude=%s\n" % ",".join(ks_repo.excludepkgs))
 
+    def _import_rpm_keys(self):
+        """Import GPG keys to RPM database."""
+        if conf.payload.default_rpm_gpg_keys:
+            arch = util.execWithCapture("uname", ["-i"]).strip().replace("'", "")
+            vers = util.execWithCapture("rpm", ["--eval", "'%{fedora}'"]).strip().replace("'", "")
```

Comment:
I'm pretty sure you can get these things from the DNF API... @dmach, @j-mracek?

Reply:
I can, but they'll give me "rawhide" instead of "34" and similar niceties, so this seemed really easier.

Reply:
Yeah, but this doesn't work either, since `%fedora` is not defined everywhere.

Reply:
Yeah, what if it is not fedora or rh?

Reply:
See https://github.com/rpm-software-management/libdnf/issues/43